packages/web-react/src/components/Flex/demo/FlexHorizontalLayout.tsx

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import DocsBox from '../../../../docs/DocsBox';
 import Flex from '../Flex';
 
-const FlexRowLayout = () => (
-  <Flex>
+const FlexHorizontalLayout = () => (
+  <Flex direction="horizontal">
```

Comment:
Horizontal is the default value, isn't it?

```suggestion
  <Flex>
```